features/steps/cocoa_steps.rb

```diff
@@ -274,8 +259,13 @@ def wait_for_true
   raise 'Assertion not passed in 30s' unless assertion_passed
 end
 
-def send_keys_to_element(element_id, text)
-  element = find_element(@element_locator, element_id)
-  element.clear()
-  element.set_value(text)
+def execute_command(action, scenario_name)
+  command = { action: action, scenario_name: scenario_name, scenario_mode: $scenario_mode }
+  Maze::Server.commands.add command
+  Maze.driver.click_element :execute_command
+  $scenario_mode = nil
+  # Ensure fixture has read the command
+  count = 100
+  sleep 0.1 until Maze::Server.commands.remaining.empty? || (count -= 1) < 1
+  raise 'Test fixture did not GET /command' unless Maze::Server.commands.remaining.empty?
```

Comment:
Did you have problems without this, or is it just belt and braces?

Reply:
This is easier to diagnose than the more generic "Assertion timed out" we'd otherwise get